src/CommunityToolkit.Datasync.Client/Paging/ConcurrentObservableCollection.cs

```diff
@@ -16,7 +16,7 @@ namespace CommunityToolkit.Datasync.Client;
 /// <typeparam name="T"></typeparam>
 public class ConcurrentObservableCollection<T> : ObservableCollection<T>
 {
-    private readonly SynchronizationContext context = SynchronizationContext.Current!;
+    private readonly SynchronizationContext context = SynchronizationContext.Current! ?? new SynchronizationContext();
```

Comment:
For whatever reason, the `SynchronizationContext.Current` was always `null` here in Uno so I added an initialization if it's null. Would be good to get to the bottom of why. 

I tried setting this somewhere in the Uno-specific code as well rather than updating the class itself. I was unsuccessful.

Reply:
I noticed some null checks down below. I'm not sure how this affects the logic of the methods that use this property. I was more focused on getting the Uno app functional.